### DISCUSSION TITLE
Daily Challenge "Minimum Effort Level"

### DISCUSSION THREAD

**Discussion by ReinIsNOTaDev** [ppy/osu#29208] [Q&A] [answered]
peppy said that there should be a minimum effort level and NF scores that miss everything will lose their combos.
![osu!_PIZ40SNPrK](https://github.com/user-attachments/assets/1bc90fcc-97b3-4aca-aa78-e716132cb567)
I set a 324 score NFDT(x2) play, but set other good passes and ended up with a 380k score, but I still lost my streak.
The next daily, Utiba set a 960 score NFDT(x?)HD play, but by the end had a 1.04M score play. Her streak is still there.

So:
What will be the minimum effort level?
Does overwritten scores matter?
Will I get my streak back?

**Comment by peppy** [comment]
There is none yet.

What's your username?

**Reply by SupDos** [reply]
Their account is https://osu.ppy.sh/users/35824254

**Reply by ReinIsNOTaDev** [reply]
^ above is correct information 😁

**Reply by ReinIsNOTaDev** [reply]
any updates?

**Reply by peppy** [reply]
You didn't have a passing score as far as I could tell.

**Reply by ReinIsNOTaDev** [reply]
https://osu.ppy.sh/multiplayer/rooms/800901?page=52#scores score should be here

**Reply by ReinIsNOTaDev** [reply]
any updates?

**Reply by ReinIsNOTaDev** [reply]
> You didn't have a passing score as far as I could tell.

i should have a passing score on every daily challenge so far, so not sure where you got that from.

**Reply by peppy** [reply, marked as answer]
We've run a recalculation on all users, which seems to have fixed your streak. Thanks for your patience.
